sources/packages/web/src/helpers/utilfunctions.ts

```diff
@@ -24,4 +26,15 @@ export default class Helper {
     const formattedDate = new Date(date);
     return formattedDate.toLocaleDateString("en-CA", options);
   }
+
+  static addressLineItemList(address: Address) {
+    return [
```

Comment:
As we talked, I believe that this method should return a simple formatted string to be directly bound to Vue template or form.io definition.